data_steward/deid/config/ids/tables/observation.json

```diff
@@ -107,12 +107,32 @@
         },
```

Comment:
Where will `:join_tablename` be coming from? I think these should be good but just want to double check there won't be an issue with the alias name.

Reply:
Yeah I think this is the actual prefix all the tables need to be using. I wasn't sure and that's probably why I left these parts untouched, and only changed the concept_id part.